dataeng/resources/setup-exporter.sh

```diff
@@ -0,0 +1,42 @@
+# Check that the DATE_MODIFIER is set to a Sunday, or if blank the current day is a Sunday
+DAYOFWEEK=$(date +%u ${DATE_MODIFIER})
+if [ $DAYOFWEEK != 7 ]; then
+   exit 1
+fi
```

Comment:
What is this about?   The job can only use data that was generated on Sundays?  I hope that job never breaks....

Reply:
I'm not sure why we have it, the CWSM dumps are only generated on sundays ?

Reply:
I wrote that because the job will run but fail to upload to S3 unless it's run on a date that's a sunday.

Reply:
The CWSM dumps are scheduled on Sundays, but if they fail on Sunday and are run on a Monday, then this prevents us from being able to change the date modifier to some other date.   I think that some new users (maybe Emilio?) when first running this specified today's date instead of the Sunday date, and I think Andrew put that in originally to deal with that.   I'm fine with it present or absent, until it becomes a problem one way or the other.